examples/all-clusters-app/nxp/mw320/binding-handler.cpp

```diff
@@ -92,8 +93,8 @@ static void BoundDeviceChangedHandler(const EmberBindingTableEntry & binding, ch
         // command (SwitchCommandHandler)
         {
             Clusters::OnOff::Commands::Toggle::Type toggleCommand;
-            Controller::InvokeCommandRequest(peer_device->GetExchangeManager(), peer_device->GetSecureSession().Value(),
-                                             binding.remote, toggleCommand, onSuccess, onFailure);
+            VerifyOrDie(exchangeMgr != nullptr && sessionHandle != nullptr);
```

Comment:
If this is the case, should we perhaps alter the `BoundDeviceChangedHandler` to always pass `sessionHandle` and `exchangeMgr` by reference?

Reply:
This is not always the case see line `mBoundDeviceChangedHandler(*iter, nullptr, nullptr, bindingContext->GetContext());` in `src/app/clusters/bindings/BindingManager.cpp`